matador/utils/chem_utils.py

```diff
@@ -643,3 +645,36 @@ def get_formula_from_stoich(stoich, elements=None, tex=False, sort=True, latex_s
                 else:
                     form += elem[0] + str(int(elem[1]))
     return form
+
+def magres_reference_shifts(
+    magres: list,
+    reference: dict,
+    shielding_key: str = "chemical_shielding_iso",
+    shift_key: str = "chemical_shift_iso",
+):
+    """ Convert chemical shielding to chemical shift.
+
+    Parameters:
+        magres (list): list of containing magres data.
+        reference (str): dictionary of conversion values in the form {element: [gradient, constant]}.
+
+    Keyword arguments:
+        shielding_key (str): the data key for which the shielding is stored under.
+        shift_key (str): the data key for which the shift will be stored under.
+    """
+
+    relevant_species = list(reference.keys())
+    shielding_key_plural = ''.join([shielding_key, 's'])
+    shift_key_plural = ''.join([shift_key, 's'])
```

Comment:
I might be missing something, but...


```suggestion
    shielding_key_plural = shielding_key + "s"
    shift_key_plural = shift_key + "s"
```

More generally, I'm not sure this even needs to be customisable. Would you ever store these under different keys?

Reply:
The only example that I can think of is that someone might want to use the anisotropic versions. In any case, I try to write my code to be flexible, but if you want to fix it to just shielding_iso and shift_iso, that's fine with me.

Reply:
Aniso is already "referenced" as it is as difference between shieldings, its just a measure of the shape of the tensor, so I think this can be removed and hardcoded